test/python/providers/test_backendconfiguration.py

```diff
@@ -110,4 +112,4 @@ def test_get_channel_prefix_index(self):
 
     def _test_lists_equal(self, actual, expected):
         """Test if 2 lists are equal. It returns ``True`` is lists are equal."""
-        return collections.Counter(actual) == collections.Counter(expected)
+        return collections.Counter(actual) == collections.Counter(expected)
```

Comment:
It seems you removed the newline at the end of file?

Reply:
My mistake, thanks for catching that....